src/fidesops/models/privacy_request.py

```diff
@@ -222,6 +231,40 @@ def cache_identity(self, identity: PrivacyRequestIdentity) -> None:
                     value,
                 )
 
+    def persist_identity(self, db: Session, identity: PrivacyRequestIdentity) -> None:
```

Comment:
This is possible, we currently don't handle identity data within that method at all, in favour of caching it separately from the model. The reason I left it that way was because we don't need the identity data in the `PrivacyRequest` table, and those ORM overrides should mainly focus on what that table needs. In the event of deletion, we do need to clear the foreign keys in order to process deleted of `PrivacyRequest`s.